app/src/main/java/com/liato/bankdroid/TransactionsActivity.java

```diff
@@ -66,7 +66,7 @@ public void onCreate(Bundle savedInstanceState) {
         viewBankName.setText(bank.getDisplayName());
         viewAccountName.setText(account.getName());
         viewAccountBalance
-                .setText(Helpers.formatBalance(account.getBalance(), account.getCurrency()));
+                .setText(Helpers.formatBalance(account.getBalance(), account.getCurrency(), false));
```

Comment:
What is your opinion on overloaded methods? Since we have a couple of places where strikethrough is `false`. Or another method name? I am fine with this solution, however.